src/common/helpers/paramValidator.js

```diff
@@ -74,15 +77,32 @@ const referencedParamValidator = async (param, model, next) => {
   const paramKey = Object.keys(param);
   const paramValue = String(Object.values(param)).toLowerCase();
   const paramExists = await models[model].findOne({
-    where: { [paramKey]: paramValue },
+    where: { [paramKey]: paramValue.trim() },
   });
 
   if (!paramExists) {
     const error = new Error(`${model} ${paramValue} does not exist`);
     error.status = 404;
     return next(error);
   }
+  return next();
+};
 
+const relationalValidator = async (params, model, next) => {
+  const paramsKey = Object.keys(params);
+  const paramsValue = Object.values(params);
+  const rowExists = await models[model].findOne({
+    where: {
+      [paramsKey[0]]: { [iLike]: paramsValue[0] },
+      [paramsKey[1]]: { [iLike]: paramsValue[1] },
+    },
+  });
+
+  if (rowExists) {
+    const error = new Error(`Topic already exists for ${paramsValue[1]}`);
```

Comment:
@Chiazokam Nice work validating input. I guess you want to use the value in `model` instead of hardcoding `Topic`

Reply:
Owhhh....thanks.
Will correct that

Reply:
Done

Reply:
It seems you have not pushed the changes

Reply:
Sorry, 'twas network. Didn't know it hadn't pushed